lib/pyld/documentloader/requests.py

```diff
@@ -55,12 +58,16 @@ def loader(url, options={}):
                     'the URL\'s scheme is not "https".',
                     'jsonld.InvalidUrl', {'url': url},
                     code='loading document failed')
-            headers = options.get('headers')
-            if headers is None:
-                headers = {
-                    'Accept': 'application/ld+json, application/json'
-                }
-            response = requests.get(url, headers=headers, **kwargs)
```

Comment:
It is unclear how this PR is changing the behavior. From what I am seeing, even before the PR a user could do

```python
requests_document_loader(…, headers={
    'Foo': 'bar',
})
```

it should be working on implementation at `master`. Or am I misunderstanding it?